src/app/databaseService.ts

```diff
@@ -55,9 +57,9 @@ async function getVehicleDetails(
   const vehicleDetailsResult = vehicleDetailsQueryResult[0][0] as VehicleQueryResult;
 
   if (
-    vehicleDetailsResult === undefined
-    || vehicleDetailsResult.id === undefined
-    || vehicleDetailsResult.result === undefined
+    vehicleDetailsResult === undefined ||
```

Comment:
Optional chaining might have been neater i.e. vehicleDetailsResult**?**.id.

Reply:
This change is just the formatter, and with this code being in production and working I don't want to refactor.